SecretManager/samples/V1/SecretManagerServiceClient/access_secret_version.php

```diff
@@ -24,8 +24,9 @@
 
 // [START secretmanager_v1_generated_SecretManagerService_AccessSecretVersion_sync]
 use Google\ApiCore\ApiException;
+use Google\Cloud\SecretManager\V1\AccessSecretVersionRequest;
 use Google\Cloud\SecretManager\V1\AccessSecretVersionResponse;
-use Google\Cloud\SecretManager\V1\SecretManagerServiceClient;
+use Google\Cloud\SecretManager\V1\Client\SecretManagerServiceClient;
```

Comment:
Again looks like a BC break?

Reply:
This is a sample... samples don't constitute a BC break